rules/filename-case.js

```diff
@@ -56,19 +56,17 @@ const cases = {
 /**
 Get the cases specified by the option.
 
-@param {unknown} context
+@param {object} config
 @returns {string[]} The chosen cases.
 */
-function getChosenCases(context) {
-	const option = context.options[0] || {};
-
-	if (option.case) {
-		return [option.case];
+function getChosenCases(config) {
```

Comment:
should name `options` 

elint calls `options`

https://eslint.org/docs/rules/yoda#options

and other rules use it as options

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/cdbb9ffa57d7c927e07c0a44c6aab8e243a626fd/rules/catch-error-name.js#L34

Reply:
In other rules it is `config` https://github.com/eslint/eslint/blob/master/lib/rules/no-magic-numbers.js#L56

Reply:
i think it is something strange in eslint code base, let's rename to options :+1: